pkg/generator/helper/helper.go

```diff
@@ -98,13 +99,21 @@ func {{.CobraCmdFuncName}}() *cobra.Command {
 			log.SetStepName(STEP_NAME)
 			log.SetVerbose({{if .ExportPrefix}}{{ .ExportPrefix }}.{{end}}GeneralConfig.Verbose)
 
+			validation, err := validation.New()
```

Comment:
let's move the validation to after line 114.
Then the `FatalHook` is registered.

In addition the error category can be set to configuration like in line 118:
`log.SetErrorCategory(log.ErrorConfiguration)`

Reply:
Fixed